<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGenerate">
    <property name="title">
     <string>Generate</string>
    </property>
    <addaction name="action3D_models"/>
    <addaction name="actionSynthetic_data"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSegmentation"/>
    <addaction name="actionProcess_Labeled_images"/>
    <addaction name="actionTrain"/>
   </widget>
   <addaction name="menuGenerate"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action3D_models">
   <property name="text">
    <string>3D models</string>
   </property>
  </action>
  <action name="actionSegmentation">
   <property name="text">
    <string>Segmentation</string>
   </property>
  </action>
  <action name="actionProcess_Labeled_images">
   <property name="text">
    <string>Process labeled images</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>Train</string>
   </property>
  </action>
  <action name="actionGenerate_synthetic_data">
   <property name="text">
    <string>Generate synthetic data</string>
   </property>
  </action>
  <action name="actionSynthetic_data">
   <property name="text">
    <string>Synthetic data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
